libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/SummaryBottomSheet.java

```diff
@@ -177,6 +181,19 @@ private void bind() {
     timeRemainingText = findViewById(R.id.timeRemainingText);
     arrivalTimeText = findViewById(R.id.arrivalTimeText);
     rerouteProgressBar = findViewById(R.id.rerouteProgressBar);
+    updateRouteOverviewImage();
+  }
+
+  private void updateRouteOverviewImage() {
+    ImageButton routeOverviewBtn = findViewById(R.id.routeOverviewBtn);
+    boolean isDarkThemeEnabled = ThemeSwitcher.isDarkThemeEnabled(getContext());
+    Drawable routeOverviewDrawable;
+    if (isDarkThemeEnabled) {
```

Comment:
Could you add a custom attribute to the nav theme so it's just added upon inflation and remove the `if else` check?